source/stylesheets/base/_feature_sections.scss

```diff
@@ -100,6 +112,10 @@
       img {
         height: 220px;
       }
+
+      .bracket {
+        @include generate-bracket(150px);
```

Comment:
So this is a case where nesting got a little weird. The output css for this is, `.featured-flavors .container .bracket`. Ideally we want it to be `.bracket`. That way it's available globally. I'd recommend creating a `_bracket.scss` in your `components` folder. The moving `.bracket { ... }` into there.